pkg/plugins/golang/v2/scaffolds/internal/templates/api/types.go

```diff
@@ -87,7 +87,7 @@ type {{ .Resource.Kind }}Status struct {
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
-{{ if not .Resource.Namespaced }} //+kubebuilder:resource:scope=Cluster {{ end }}
+{{ if not .Resource.Namespaced }}//+kubebuilder:resource:scope=Cluster{{ end }}
```

Comment:
Nit: make scaffolding changes in a separate PR

Reply:
Tests will break without this. These spaces were removed by a `go fmt` call that is no longer used.